lib/sda/local.go

```diff
@@ -173,7 +179,11 @@ func (l *LocalTest) NewNode(tn *TreeNode, protName string) (*Node, error) {
 		TreeNodeID:   tn.Id,
 		RoundID:      uuid.NewV4(),
 	}
-	return NewNode(o, tok)
+	node, err := NewNode(o, tok)
+	if err == nil {
+		l.Nodes = append(l.Nodes, node)
+	}
+	return node, err
```

Comment:
but why should we want to do that ... ? I'm sorry for nitpicking but for me it does not make sense to test having a Node non-registered to an Overlay since it will never happen in practice ... ?
But OK, it's for local test anyway.